src/components/Button.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class Button extends React.Component {
+  render() {
+    return <button>{this.props.buttonName}</button>;
+  }
+}
+
```

Comment:
`buttonName` should be string, enforce this with `propTypes` [see [requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/button/validateAsString.png)]